mk/config.mk

```diff
@@ -304,3 +304,8 @@ CFG_DYN_SHM_CAP ?= y
 # Enables support for larger physical addresses, that is, it will define
 # paddr_t as a 64-bit type.
 CFG_CORE_LARGE_PHYS_ADDR ?= n
+
+# Define the max bit size for big number representation in TEE priviledge
+# layer. 4096 is currently the max supported by the underlying crypto library.
+# TEEs running in constrained sized memory may want to optimize this value.
+CFG_CORE_BIGNUM_MAX_BITS ?= 4096
```

Comment:
- s/priviledge/privileged/
- The crypto library (LibTomCrypt) is a user of the big numbers library, so it is not really "underlying", it is the opposite. The key sizes are limited by the bignum size.

How about:

```
# Define the maximum size, in bits, for big numbers in the TEE core (privileged layer).
# This value is an upper limit for the key size in any cryptographic algorithm
# implemented by the TEE core.
# Set this to a lower value to reduce the memory footprint.
```

Reply:
thanks for the suggestion, I agree with your proposal.